.processes/bounty.md

```diff
@@ -68,6 +68,14 @@ After a bounty is [reviewed](#creating-a-bounty), two dates need to be set:
 
 The deadline set for the bounty hunter should be the `pickup deadline`, this way, it allows the tech team to fully avoid working on the task in the case it's completed or assist the bounty hunter on completing it. Either way, bounty hunter must be showing good progress throughout the time they are accepted to work on it.
 
+## Creating a PR
+
+In order to ensure that external contributions pass our CI tests, the bounty creator must:
+
+1. Create a branch prefixed by `bounty/<name>` and push it.
+2. Bounty hunter may create PRs that merge into that branch.
+3. When work is complete, PR is merged into the branch, ensuring tests pass.
```

Comment:
```suggestion
3. When work is complete, PR is merged into the branch, ensuring tests pass.
4. A member of the development team approves the PR and merges it to `master`
```

Reply:
This should complete the workflow

Reply:
This is described in the `Awarding the prize` section, this section is only meant to describe how to create a bounty PR